contrib/!irc/erc/README.md

```diff
@@ -26,6 +26,8 @@ Layer for [ERC IRC chat][].
   (using [erc-view-log][])
 - YouTube videos Thumbnails inline (using [erc-yt][])
 - Social Graph for ERC messages (using [erc-social-graph][])
+- ERC yank, if pasting more than 5 lines, create a gist for it. (needs to have
+  `git-enable-github-support` set to `t` in the git layer)
```

Comment:
Ok, @syl20bnr, fixed it and updated to latest commit. :)

On Tue, Jun 9, 2015 at 10:41 PM, Sylvain Benner <notifications@github.com>
wrote:

> In contrib/!irc/erc/README.md
> <https://github.com/syl20bnr/spacemacs/pull/1887#discussion_r32086046>:
>
> > @@ -26,6 +26,8 @@ Layer for [ERC IRC chat][].
> >    (using [erc-view-log][])
> >  - YouTube videos Thumbnails inline (using [erc-yt][])
> >  - Social Graph for ERC messages (using [erc-social-graph][])
> > +- ERC yank, if pasting more than 5 lines, create a gist for it. (needs to have
> > +  `git-enable-github-support` set to `t` in the git layer)
>
> This variable does not exist anymore, all the github related packages are
> now in the github layer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1887/files#r32086046>.
>